build.sbt

```diff
@@ -78,12 +78,7 @@ lazy val sharedSettings = MimaSettings.settings ++ scalaVersionSettings ++ Seq(
   // don't use fatal warnings in tests
   scalacOptions in Test ~= (_ filterNot (_ == "-Xfatal-warnings")),
 
-  mimaPreviousArtifacts := (
-    if (CrossVersion isScalaApiCompatible scalaVersion.value)
-      Set("org.scalacheck" %%% "scalacheck" % "1.14.0")
-    else
-      Set.empty
-  ),
+  mimaPreviousArtifacts := Set("org.scalacheck" %% "scalacheck" % "1.14.0"),
```

Comment:
MiMa is failing for the Scalajs and Scala native builds in similar ways:

```
[info] scalacheck: found 2 potential binary incompatibilities while checking
[info] against org.scalacheck:scalacheck_2.13.0-M3:1.14.0 
[error]  * class org.scalacheck.Platform#EnableReflectiveInstantiation
[error]    does not have a correspondent in current version 
[error]    filter with:
[error]    ProblemFilters.exclude[MissingClassProblem]("org.scalacheck.Platform$EnableReflectiveInstantiation")
[error]  * method deserializeTask(java.lang.String,scala.Function1)org.scalacheck.ScalaCheckRunner#BaseTask
[error]    in class org.scalacheck.ScalaCheckRunner has a different result
[error]    type in current version, where it is sbt.testing.Task rather than
[error]    org.scalacheck.ScalaCheckRunner#BaseTask 
[error]    filter with:
[error]    ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.scalacheck.ScalaCheckRunner.deserializeTask")
```

I assumed Mima supports those platforms, but maybe it doesn't?  Or is there more sbt magic needed for it to work?